src/components/StyledText.js

```diff
@@ -3,9 +3,11 @@ import {
   Text
 } from 'react-native'
 import EStyleSheet from 'react-native-extended-stylesheet'
+import { append } from 'ramda'
 
 const StyledText = (props) => {
-  const textStyle = ( props.textStyle ? [styles.defaultText, styles[props.textStyle]] : styles.defaultText )
+  var textStyle = ( props.textStyle ? [styles.defaultText, styles[props.textStyle]] : [styles.defaultText] )
```

Comment:
Avoid using `var` here. Use `let` when declaring a variable that will later be reassigned, per ES6 standards (unless RN restricts you from doing it for some reason).